packages/eas-cli/src/commands/build/index.ts

```diff
@@ -63,6 +65,9 @@ export default class Build extends EasCommand {
       default: false,
       description: 'Run build locally [experimental]',
     }),
+    output: Flags.string({
+      description: 'Output path for local build.',
```

Comment:
```suggestion
      description: 'Output path for local build',
```
We should check all the descriptions and normalize them (not in this PR).